ci/pipelines/pr.yml

```diff
@@ -1043,14 +1044,67 @@ jobs:
     - <<: *put-pact-provider-success-status
       put: adminusers-pull-request
 
+  - <<: *job-definition
+    name: adminusers-consumer-pact-test
+    plan:
+    - <<: *get-ci
+    - <<: *get-pull-request
+      resource: adminusers-pull-request
+    - task: run-consumer-pact-tests
+      privileged: true
+      config:
+        container_limits: {}
+        platform: linux
+        image_resource:
+          type: registry-image
+          source:
+            repository: govukpay/concourse-runner
+        inputs:
+          - name: src
+        outputs:
+          - name: pacts
+        run:
+          path: bash
+          dir: src
+          args:
+            - -ec
+            - |
+              source /docker-helpers.sh
+              start_docker
+
+              function cleanup {
+                echo "CLEANUP TRIGGERED"
+                clean_docker
+                stop_docker
+                echo "CLEANUP COMPLETE"
+              }
+
+              trap cleanup EXIT
+
+              commit="$(cat .git/resource/head_sha)"
+              pr="$(cat .git/resource/pr)"
+              mvn clean package \
+                        -DPACT_BROKER_URL=https://pact-broker-test.cloudapps.digital \
+                        -DPACT_BROKER_USERNAME="" \
+                        -DPACT_BROKER_PASSWORD="" \
+                        -DPACT_CONSUMER_VERSION="${commit}" \
+                        -DPACT_CONSUMER_TAG="${pr}" \
+                        -Dpact.provider.version="${commit}" \
+                        -DrunConsumerContractTests=true
```

Comment:
I'm puzzled as to why this works as there is no runConsumerContractTests profile in adminusers's pom.xml

Reply:
Ah I've added it in the adminusers PR - I guess probably the adminusers PR needs to come first to add the tests to be run before concourse can run them!

Reply:
Also maybe not needed for this PR but if you have time in the future it'd be good to extract this code block as it appears to look like https://github.com/alphagov/pay-ci/blob/master/ci/pipelines/pr.yml#L1417-L1445

Reply:
Ah yeh good point, definitely worth doing